presto-tpch/src/test/java/io/prestosql/plugin/tpch/TestTpchMetadata.java

```diff
@@ -129,6 +130,17 @@ public void testTableStatsWithConstraints()
         });
     }
 
+    @Test
+    public void testHiddenSchemas()
```

Comment:
- Add tests for tpcdsmetadata too.
- Could you please add an explicit test for `SHOW TABLES FROM ..` and maybe `USE ... ; SHOW TABLES;`?